runtimes/cmake/Modules/HandleLitArguments.cmake

```diff
@@ -0,0 +1,20 @@
+
+macro(serialize_lit_param param value)
+  string(APPEND SERIALIZED_LIT_PARAMS "config.${param} = ${value}\n")
```

Comment:
This looks reasonable, but could we avoid using the global variable `SERIALIZED_LIT_PARAMS` here? We should thread it through the functions instead, otherwise these functions all "return their result" by modifying the global variable, which is yucky.